kimchi/src/verifier.rs

```diff
@@ -37,24 +37,24 @@ where
     EFqSponge: Clone + FqSponge<BaseField<G>, G, ScalarField<G>>,
 {
```

Comment:
Question here: `ResultOracle` is only being used inside our verifier. I wonder why this is a public struct at all. What is the use of this struct outside? If it has none, because we are only considering our "official" verifier, then the fields `digest` and `combined_inner_product` are never used in our verifier.